modules/govuk_jenkins/templates/jobs/search_api_learn_to_rank.yaml.erb

```diff
@@ -0,0 +1,91 @@
+---
+- scm:
+    name: search-api
+    scm:
+      - git:
+          url: git@github.com:alphagov/search-api.git
+          branches:
+            - migrate-ltr-to-jenkins
+- job:
+    name: search-api-learn-to-rank
+    display-name: Search API Learn to Rank
+    project-type: freestyle
+    description: This job spins up an EC2 instances to generate training data, then uses AWS SageMaker to train and deploy a model.
+    scm:
+      - search-api
+    logrotate:
+        numToKeep: 100
+    triggers:
+      - timed: |
+          TZ=Europe/London
+          0 22 * * SUN
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+          artifact-num-to-keep: 5
+    publishers:
+      - trigger-parameterized-builds:
+          - project: Success_Passive_Check
+            condition: SUCCESS
+            predefined-parameters: |
+                NSCA_CHECK_DESCRIPTION=<%= @service_description %>
+                NSCA_OUTPUT=<%= @service_description %> success
+          - project: Failure_Passive_Check
+            condition: FAILED
+            predefined-parameters: |
+                NSCA_CHECK_DESCRIPTION=<%= @service_description %>
+                NSCA_OUTPUT=<%= @service_description %> failed
+    parameters:
+      - string:
+          name: AWS_ROLE_ARN
+          description: The ARN for the AWS Role to assume
+          default: <%= @aws_role_arn %>
+      - string:
+          name: ELASTICSEARCH_URI
+          description: The URI of Elasticsearch cluster for generating training data
+          default: <%= @elasticsearch_uri %>
+      - string:
+          name: IMAGE
+          description: The image name to use in SageMaker for training the model
+          default: <%= @image %>
+      - string:
+          name: TRAIN_INSTANCE_TYPE
+          description: The type of instance used by SageMaker to train the model
+          default: <%= @train_instance_type %>
+      - string:
+          name: TRAIN_INSTANCE_COUNT
+          description: The number instances used by SageMaker to train the model
+          default: <%= @train_instance_count %>
+      - string:
+          name: DEPLOY_INSTANCE_TYPE
+          description: The type of instance used by SageMaker to host the deployed model
+          default: <%= @deploy_instance_type %>
+      - string:
+          name: DEPLOY_INSTANCE_COUNT
+          description: The number instances used by SageMaker to host the deployed model
+          default: <%= @deploy_instance_count %>
+      - string:
+          name: GOVUK_ENVIRONMENT
+          default: <%= @environment %>
+    builders:
+      - shell: |
+          set -ex
+
+          echo ${SSH_PRIVATE_KEY} | sed -E 's/(-+(BEGIN|END) OPENSSH PRIVATE KEY-+) *| +/\1\n/g' > ./ssh_private_key.pem
```

Comment:
Nice :) one other approach you could take is to base64 encode/decode the key when passing it around (not suggesting changing it here though, this is fine with me 👍)